Sources/SentryCrash/Recording/SentryCrashReportStore.c

```diff
@@ -212,6 +212,13 @@ sentrycrashcrs_readReport(int64_t reportID)
     return result;
 }
 
+void
+sentrycrashcrs_screenShotPath(int64_t reportID, char *pathBuffer)
+{
+    snprintf(pathBuffer, SentryCrashCRS_MAX_PATH_LENGTH, "%s/%s-report-%016llx", g_reportsPath,
+        g_appName, reportID);
```

Comment:
`m`: Should we maybe also wrap this with `pthread_mutex_lock(&g_mutex);` and `pthread_mutex_unlock(&g_mutex);`?

Reply:
Im not sure. The method to get the report file path does not use it